<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteEdtorWindow</class>
 <widget class="QMainWindow" name="PaletteEdtorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор палитры</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Число цветов палитры</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Тип палитры</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>2 байта RGB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 байта RGGB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 байта RGB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 байта RGB0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 байта RGBF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 байта RGB с прозрачностью</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Просмотр палитры</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>385</width>
            <height>105</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="margin">
            <number>3</number>
           </property>
           <item>
            <widget class="PaletteViewer" name="paletteWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="menu_new_action"/>
    <addaction name="separator"/>
    <addaction name="menu_save_action"/>
    <addaction name="menu_open_action"/>
    <addaction name="separator"/>
    <addaction name="menu_exit_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new_action"/>
   <addaction name="save_action"/>
   <addaction name="open_action"/>
   <addaction name="separator"/>
   <addaction name="exit_action"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="new_action">
   <property name="text">
    <string>Новая палитра</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="menu_new_action">
   <property name="text">
    <string>Новая палитра</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="menu_save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="menu_open_action">
   <property name="text">
    <string>Загрузить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="menu_exit_action">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X, Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaletteViewer</class>
   <extends>QWidget</extends>
   <header>paletteviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>menu_save_action</sender>
   <signal>triggered()</signal>
   <receiver>save_action</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_open_action</sender>
   <signal>triggered()</signal>
   <receiver>open_action</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_new_action</sender>
   <signal>triggered()</signal>
   <receiver>new_action</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_exit_action</sender>
   <signal>triggered()</signal>
   <receiver>exit_action</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_action</sender>
   <signal>triggered()</signal>
   <receiver>PaletteEdtorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
